### ISSUE TITLE
color_gradient has "if vector" 

### ISSUE THREAD

**Issue by pinq-** [Zulko/moviepy#1205]
```
half_mask = color_gradient((2000,2000),p1=(0,200), p2=(100,200))
```
#### Expected Behavior
make the color gradient

#### Actual Behavior
get error:
```
moviepy\video\tools\drawing.py", line 147, in color_gradient
    if vector:
ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
```

#### Steps to Reproduce the Problem
Run `half_mask = color_gradient((2000,2000),p1=(0,200), p2=(100,200))` line. 

#### Specifications

  - Python Version: 3.6.5
  - Moviepy Version: 1.0.3
  - Platform Name: Windows
  - Platform Version: 10

**Comment by tburrows13** [comment]
Thanks for the report. I have successfully reproduced this error.

**Comment by tburrows13** [comment]
You should be able to fix it by editing the line `if vector:` to say `if vector is not None:`.

Let me know if this fixes it, and feel free to open a pull request with the change if you like. If not, I'll do it.

**Comment by pinq-** [comment]
looks like that works. I don't know how to make pull request, so you can do it :)

**Comment by tburrows13** [comment]
Fixed in #1212 :)
